src/openrct2/scripting/bindings/ride/ScTrackSegment.cpp

```diff
@@ -29,13 +29,43 @@ void ScTrackSegment::Register(duk_context* ctx)
     dukglue_register_property(ctx, &ScTrackSegment::type_get, nullptr, "type");
     dukglue_register_property(ctx, &ScTrackSegment::description_get, nullptr, "description");
     dukglue_register_property(ctx, &ScTrackSegment::elements_get, nullptr, "elements");
-    dukglue_register_property(ctx, &ScTrackSegment::beginZ_get, nullptr, "beginZ");
     dukglue_register_property(ctx, &ScTrackSegment::beginDirection_get, nullptr, "beginDirection");
+    dukglue_register_property(ctx, &ScTrackSegment::endDirection_get, nullptr, "endDirection");
+    dukglue_register_property(ctx, &ScTrackSegment::beginAngle_get, nullptr, "beginAngle");
+    dukglue_register_property(ctx, &ScTrackSegment::endAngle_get, nullptr, "endAngle");
```

Comment:
I'd much rather they were exposed as yaw, roll, pitch rather than direction, bank and angle. yaw, roll, pitch are much clearer as to what they mean compared to direction, bank and angle.

Reply:
They were named this by whoever added them... I haven't done git-blame to see who. I will change them.

Reply:
Please don't change the names already exposed to the plugin developers. It would be a breaking change that would break existing plugins.